class IntentionalError(Exception):
    pass


raise IntentionalError()
